mobile/test/trade_test.dart

```diff
@@ -128,13 +129,18 @@ void main() {
 
     WalletChangeNotifier walletChangeNotifier = WalletChangeNotifier(walletService);
 
+    PositionChangeNotifier positionChangeNotifier = PositionChangeNotifier(positionService);
+
+    // We have to have current price, otherwise we can't take order
+    positionChangeNotifier.price = Price(bid: 30000.0, ask: 30000.0);
```

Comment:
nit: I think could potentially solve this cleaner by adding an explicit `Extension` to the `PositionChangeNotifier` in the tests that adds a setter so we can set the price; then we don't have to expose the private `_price`.

Reply:
I don't think this adds anything, a direct getter and setter `==` a public member....